hoomd/md/NeighborListStencil.cc

```diff
@@ -325,5 +325,6 @@ void export_NeighborListStencil(py::module& m)
     {
     py::class_<NeighborListStencil, NeighborList, std::shared_ptr<NeighborListStencil> >(m, "NeighborListStencil")
         .def(py::init< std::shared_ptr<SystemDefinition>, Scalar, Scalar, std::shared_ptr<CellList>, std::shared_ptr<CellListStencil> >())
```

Comment:
```suggestion
        .def(py::init< std::shared_ptr<SystemDefinition>, Scalar)
```
No need to pass these in from Python. The C++ code manages them internally.